src/nl/surf/eduhub_rio_mapper/api.clj

```diff
@@ -111,6 +118,7 @@
 
 (defn make-app [{:keys [auth-config clients] :as config}]
   (-> routes
+      (wrap-uuid-validator)
```

Comment:
This breaks to `/status/:token` endpoint because it does not have an `:id` parameter.  Before fixing this, make a test for the status endpoint (and proof my assumption) to avoid introducing regression like this in the future.